src/ProofOfConcepts/ConnectorMessageDrawer/DrawerGrid/ConnectorDrawerMessageStatistics.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { FunctionComponent } from "react";
+
+import {
+  Alert,
+  Card,
+  CardBody,
+  CardFooter,
+  Divider,
+  Flex,
+  FlexItem,
+  TextContent,
+  TextList,
+  TextListVariants,
+  TextListItem,
+  TextListItemVariants,
+  Title,
+  TitleSizes,
+  CardTitle,
+  Grid,
+  GridItem,
+} from "@patternfly/react-core";
+import { ExclamationIcon } from "@patternfly/react-icons";
+import CheckIcon from "@patternfly/react-icons/dist/esm/icons/check-icon";
+
+import "./ConnectorDrawer.css";
+
+export const ConnectorDrawerMessageStatistics = () => {
+  return (
+    <Flex direction={{ default: "column" }}>
```

Comment:
Wonder, why Flex and not [Stack](https://patternfly-react.surge.sh/layouts/stack) out of curiousity?

Reply:
I actually had not considered stack, I started with flex and it seemed to be working